Components/Header.js

```diff
@@ -0,0 +1,19 @@
+import { data } from '../data.js';
+
+export function createHeader() {
+  const { header: {titleTeam, descriptionTeamTitle, descriptionTeamText} } = data; 
+  const headerElement = document.createElement('header');
+  headerElement.className = 'header';
+
+  const titleCrew = document.createElement('h1');
+  titleCrew.className = 'titleTeam';
+  titleCrew.textContent = titleTeam;
```

Comment:
I'd avoid using constant names so linked to the content of the website. It could be better `headerTitle`, `teamTitle`, and so on.